src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -316,7 +316,13 @@ protected Task ResolveAuthorityAsync()
 
         protected KeyValuePair<string, string>? GetCcsUpnHeader(string upnHeader)
```

Comment:
Can we return null instead? It will a tiny bit faster. Same for line 314 above. Otherwise there's no point in returning a nullable struct, if you're not actually returning null.